### ISSUE TITLE
Interpolation in properties

### ISSUE THREAD

**Issue by sandrojohanides** [SublimeText/Sass#54]
Couple of problems with this:
- the `#{nth()}` interpolation color with the function inside
- the `-` hyphen color after and before the interpolation
- the `-` minus and `/` divide color in value after interpolation

![screen shot 2018-12-10 at 13 49 20](https://user-images.githubusercontent.com/8327913/49735782-84e2ad80-fc88-11e8-9a13-ffe3a877dbe2.png)

```scss
	#{nth($dirs, 1)}: 100%; // `bottom: 100%` by default
	#{nth($dirs, 2)}: $offset; // `left: 50%` by default

	height: 0;
	width: 0;
	margin: auto;
	margin-#{nth($dirs, 2)}: (-$size / 2);

	border: solid transparent;
	border-#{nth($dirs, 1)}-color: $color;
	border-width: ($size / 2);
```

**Comment by braver** [comment]
Right now it only supports interpolation of variables, not of full expressions. So, this issue is correct and support for sass script expressions should be added here. Thanks!

**Comment by braver** [comment]
closed via #96
